### DISCUSSION TITLE
.exe file

### DISCUSSION THREAD

**Discussion by Maxyuna** [xenesty/AkashaScanner#51] [Q&A] [answered]
Hello!
I've been wracking my brain trying to launch this for days, and the exe file that launches the program is nowhere to be found in my folder. I download/extract the file, but I literally can't launch it from anywhere. The exe file that launches the program simply doesn't exist. This probably is a dumb question and prob has an easy fix, but I'm getting really desperate at this point haha
Thank you! <3

**Comment by akrios-d** [comment]
Hello, the exe should be AkashaScanner.exe, maybe when unzipping something is removing it. Also, try to use my fork if it's possible, due to some issues in parsers about characters and artefact, currently version does(0.4.2/this repository) does not work. Also if need any help or find bugs please report in my fork

**Reply by Maxyuna** [reply, marked as answer]
Hello, thank you so much for the answer!!
I was able to figure out a workaround after a few days of trying out, basically I downloaded an earlier version (the version before the newest one) and it worked perfectly, the exe file was there and I was able to launch akasha and import/export the files!
I wonder if it removes the exe file while downloading or it flags it as unsafe or something, but this works perfectly for now. It seems like it's a problem with that newest version specifically
I'll try out your fork and see if that works tho, again thank you for your reply and I hope this helps someone out there that has the same problem (:
